cves/kernel/CVE-2021-38200.yml

```diff
@@ -320,8 +336,14 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=60b7ed54a41b550d50caf7f2418db4a7e75b5bdc
+    Reported-by: Christophe Leroy <christophe.leroy@csgroup.eu>
+    Signed-off-by: Athira Rajeev <atrajeev@linux.vnet.ibm.com>
+    Tested-by: Christophe Leroy <christophe.leroy@csgroup.eu>
+    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
```

Comment:
The actual names aren't all that useful to the reader - who are they in the context of this vulnerability? Some observations: the reporter and tester are the same, and it was signed off by two different people with different affiliations (assuming from the email).